docs/welcome/what-is-tartiflette.md

```diff
@@ -0,0 +1,29 @@
+---
+id: what-is-tartiflette
+title: What is Tartiflette
+sidebar_label: What is Tartiflette?
+---
+
+Welcome, glad to see you here.
+
+**Tartiflette** is a GraphQL Engine, built on top of Python 3.6 and higher. Focus on building GraphQL API based on the **Schema Definition Language** _(commonly called 'SDL')_.
+
+## What is GraphQL?
+
+> GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data. GraphQL provides a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+
+Source: [https://graphql.org/](https://graphql.org/)
+
+## Shoud I use Tartiflette only?
+
+Our engine respect the [GraphQL Specification](https://facebook.github.io/graphql/), which doesn't specify any transport layer. You are able to integrate the engine whenever you want, and use any transport layer you want, JSON RPC, grpc or obvisouly, http.
+
+In addition to the Tartiflette Engine and based on common integration of GraphQL, we developed an GraphQL HTTP library, which use aiohttp to expose the tartiflette engine through HTTP. 
```

Comment:
In addition to the Tartiflette Engine, and based on common integrations of GraphQL, we developed a GraphQL HTTP Library, which uses `aiohttp` to expose the tartiflette engine through HTTP.